Tests/FluentAssertions.Specs/Streams/BufferedStreamAssertionSpecs.cs

```diff
@@ -0,0 +1,107 @@
+﻿#if NETCOREAPP2_1_OR_GREATER || NETSTANDARD2_1
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Sdk;
+#endif
+
+namespace FluentAssertions.Specs.Streams
+{
+    public class BufferedStreamAssertionSpecs
+    {
+#if NETCOREAPP2_1_OR_GREATER || NETSTANDARD2_1
+
+        #region HaveBufferSize / NotHaveBufferSize
+
+        [Fact]
+        public void When_asserting_a_stream_should_have_buffer_size_with_the_same_value_it_should_succeed()
```

Comment:
🔧 I recommend removing the `asserting` part to keep the scenarios more concise. E.g. `When_the_buffer_size_has_the_expected_value_it_should_succeed`

Reply:
All 56 tests have new names 🤡